melodic/distribution.yaml

```
@@ -5578,6 +5578,15 @@ repositories:
       version: mrpt-1.5
     status: maintained
   mrpt2:
+    doc:
+      type: git
+      url: https://github.com/MRPT/mrpt.git
```

```suggestion
      url: https://github.com/mrpt/mrpt.git
```

(this is a really small nit, as github is case-insensitive, but I think it would be better for it to be consistent)